app/routes.js

```diff
@@ -19,7 +19,7 @@ router.use(function (req, res, next) {
 
  res.locals.formQuery = res.locals.formQuery.slice(0,-1);
 
- if (!req.session.userName && req.originalUrl !== '/login' && req.originalUrl !== '/send-login') {
+ if (!req.session.userName && req.originalUrl !== '/login' && req.originalUrl !== '/send-login'  && req.originalUrl !== '/') {
    res.redirect('/login');
```

Comment:
we could use ['/login', '/send-login ... ].indexOf(req.originalUrl) if it wasn't for IE8